docs/gettinginvolved/infrastructure/maintaining-cloudformation.rst

```diff
@@ -34,7 +34,10 @@ Generate CloudFormation JSON template:
 
 .. prompt:: bash #
 
-   admin/create-cloudformation-template > /tmp/flocker-cluster.cloudformation.json
+   admin/create-cloudformation-template \
+       --client-ami-map-body="$(< $PWD/docker.json)" \
```

Comment:
Sorry, I should have made that clear.

The JSON maps will be generated by other commands which I am in the process of documenting in `cloudformation-release-process-FLOC-3945-2`.

All should become clear when I put that branch up for review.